planner: add doc for `tidb_ignore_inlist_plan_digest`. (#20267)
pingcap/docs-cn: system-variables.md

- 这个变量用来控制用户会话中事务的空闲超时。当用户会话处于事务状态且空闲时间超过该变量设定的值时，会话会被 Kill 掉。用户会话空闲是指没有正在执行的请求，处于等待请求的状态。
- 默认值 `0` 表示没有时间限制。

### `tidb_ignore_inlist_plan_digest` <span class="version-mark">从 v7.6.0 版本开始引入</span>

- 作用域：GLOBAL
- 是否持久化到集群：是
- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
- 类型：布尔型
- 默认值：`OFF`
- 这个变量用来控制 TiDB 在生成执行计划摘要 (Plan Digest) 时，是否忽略不同查询中 `IN` 列表的元素差异。

>     - 当为默认值 `OFF` 时，TiDB 在生成执行计划摘要时，会忽略 `IN` 列表中的元素差异（包括元素数量差异）。

Author: ```suggestion
    - 当为默认值 `OFF` 时，TiDB 在生成执行计划摘要时，不会忽略 `IN` 列表中的元素差异（包括元素数量差异）。`IN` 列表的元素差异将导致生成的执行计划摘要不同。
```

>     - 当设置为 `ON` 时，TiDB 会将 `IN` 列表中的元素（包括元素数量）作为执行计划摘要的一部分。`IN` 列表的元素差异将导致生成的执行计划摘要不同。

Author: ```suggestion
    - 当设置为 `ON` 时，TiDB 会忽略 `IN` 列表中的元素（包括元素数量）作为执行计划摘要的一部分，使用 `...` 代替其中元素，最终生成的相同的执行计划摘要。
```

- 单位：字节
- 这个变量用来控制 [Statement Summary Tables](/statement-summary-tables.md) 和 [TiDB Dashboard](/dashboard/dashboard-intro.md) 中显示的 SQL 字符串长度。

### `tidb_ignore_inlist_plan_digest` <span class="version-mark">从 v7.6.0 版本开始引入</span>

- 作用域：GLOBAL
- 是否持久化到集群：是
- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
- 类型：布尔型
- 默认值：`OFF`

> - 这个变量用来控制是否忽略 `IN` 列表中元素差异对计划摘要的影响。设置为 `ON` 时，TiDB 将考虑 `IN` 列表中元素差异（包括数量）对计划摘要的影响。设置为 `OFF` 时，TiDB 将不考虑 `IN` 列表中元素差异（包括数量）对计划摘要的影响。

Reviewer: ```suggestion
- 这个变量用来控制 TiDB 在生成执行计划摘要 (Plan Digest) 时，是否忽略不同查询中 `IN` 列表的元素差异。

    - 当为默认值 `OFF` 时，TiDB 在生成执行计划摘要时，会忽略 `IN` 列表中的元素差异（包括元素数量差异）。
    - 当设置为 `ON` 时，TiDB 会将 `IN` 列表中的元素（包括元素数量）作为执行计划摘要的一部分。`IN` 列表的元素差异将导致生成的执行计划摘要不同。
```
